app/bundles/ScoringBundle/Controller/ScoringCategoryController.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
Because these forms are so short, let's make these behave like Categories in the admin menu where forms are modal based.

Reply:
I agree it would be nice to have but we don't have any time budget remaining to change the UI this way. 

Could maybe be a UI refinement  in a future version ?